ui/lib/providers.js

```diff
@@ -0,0 +1,37 @@
+export const providers = [{
+  name: 'Okta',
+  kind: 'okta',
+  available: true,
+  validate: {
+    url: url => {
```

Comment:
is this enough to check if the url is valid? I think it is possible to customize the okta URL domain (https://developer.okta.com/docs/guides/custom-url-domain/main/). wondering In that case how are we going to handle that?